src/sagas/wallet.js

```diff
@@ -346,13 +349,13 @@ export function* loadTokens() {
 
   // Fetch all tokens, including the ones that are not registered yet
   const allTokens = yield call([wallet, wallet.getTokens]);
-  const registeredTokens = yield getRegisteredTokensUids(wallet, true);
+  const registeredTokens = yield call(tokensUtils.getRegisteredTokens, wallet);
```

Comment:
You were using the parameter `true` to exclude the default token but now you are using `false` (the default value), is this a bug or it was a bug before?

Reply:
Before we were translating the behavior of the old wallet, which only requested the custom tokens since the first screen would load the history of the default token (or HTR).

We do not need to skip the default token (or HTR) here since we started using this list to set the registered tokens and we do not have any issue loading the HTR history before we load the first wallet screen.